### PR TITLE
Landing page for event

### PR DESCRIPTION
**Pull request by wfortin**
Basic landing page with JS styling.

### CODE REVIEW

**Comment by wfortin** [comment]
Sneak preview:
image

File: index.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+<head>
+
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <title>Grayscale - Start Bootstrap Theme</title>
```
Comment (osylvain) [inline index.html]: Might wanna change this

File: index.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+                </p>
+            </div>
+        </div>
+    </section>
+
+    <div id="map"></div>
+
+    <footer>
+        <div class="container text-center">
+            <p>Some cheesy line like made with love in Quebec City</p>
```
Comment (osylvain) [inline index.html]: might want to change this

**Comment by osylvain** [comment]
I understand this is a first draft so you're probably already aware of my comments. Other than that i'd say good work man! Ship it!

File: index.html (hunk 2 above)
Comment (wfortin) [inline index.html]: Haha, yes i just needed placeholder text.

**Comment by wfortin** [comment]
Applied your comments @osylvain.

**Comment by wfortin** [comment]
Stuff that still needs to be done :
* Google Map appears to be broken with the coordinates i've entered
* Use cdn for all external libs/css
* Add speakers/talks section
* Link/embed eventbrite event

**Pull request merged**